hazelcast/src/main/java/com/hazelcast/spi/OperationFactory.java

```diff
@@ -26,9 +26,8 @@
  * {@link OperationService} uses this factory to create {@link Operation}s for each partition by calling
  * {@link #createOperation()}
  * </p>
- *
  */
-public interface OperationFactory extends DataSerializable {
```

Comment:
Yeah, I meant we're stuck with the implementations. Didn't notice that Peter's comment was on the top-level class.